src/platform/online/codeforces.py

```
@@ -51,21 +51,32 @@ class Codeforces(CompetitivePlatform):
 
+    @classmethod
+    def _api(cls, api: str, **kwargs) -> dict:
+        """传递参数构造 payload，添加首尾下划线可避免与关键词冲突"""
+        url = cls._decode_api_url(api, **kwargs)
+        json_data = fetch_url_json(url)
         return json_data['result']
 
+    @classmethod
+    def _api_with_check(cls, api: str, **kwargs) -> dict | None:
+        """
+        调用 api，并检查请求是否 OK
```

Ruff (0.13.1)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)